src/panels/lovelace/common/entity/turn-on-off-entity.ts

```diff
@@ -17,6 +17,9 @@ export const turnOnOffEntity = (
     case "cover":
       service = turnOn ? "open_cover" : "close_cover";
       break;
+    case "button":
```

Comment:
And "input_button"?

Reply:
Good point => https://github.com/home-assistant/frontend/pull/11350